app/models/ExportsDeclaration.scala

```diff
@@ -40,6 +40,7 @@ import services.DiffTools.{combinePointers, compareDifference, compareIntDiffere
 case class ExportsDeclaration(
   id: String,
   declarationMeta: DeclarationMeta,
+  eori: String = "", // Included for parity with BE model only
```

Comment:
Why do we need to default to empty string? :thinking:

Reply:
I thought that would make more sense given that we aren't using the field in the FE instead of pulling the actual EORI into it.